src/main/resources/snippets/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: serving.knative.dev/v1 # Current version of Knative
+kind: Service
+metadata:
+  name: "add service name"
```

Comment:
I think we should have `namespace` property there, as Schema validator complains about it:
<img width="634" alt="Screenshot 2021-03-17 at 15 10 19" src="https://user-images.githubusercontent.com/929743/111472706-e846ca00-8732-11eb-8d27-dda873e93b37.png">